media/CircleEditor/type.js

```diff
@@ -28,14 +28,12 @@ let typeName = {
     'DataFormat': ['CHANNELS_LAST', 'CHANNELS_FIRST'],
 };
 
-let tensorType = [
-    'FLOAT32', 'FLOAT16', 'INT32', 'UINT8', 'INT64', 'BOOLEAN', 'INT16', 'INT8', 'FLOAT64',
-    'UINT64', 'UINT32'
-];
+var tensorType =
+    ['FLOAT32', 'FLOAT16', 'INT32', 'UINT8', 'INT64', 'BOOLEAN', 'INT16', 'INT8', 'FLOAT64'];
 
-let customType = ['int', 'boolean', 'string'];
+var customType = ['int', 'boolean', 'string'];
 
-let builtinOperatorType = {
+var builtinOperatorType = {
```

Comment:
This may be too.... T.T

```suggestion
var builtinOperatorTypeEnum = {
```

Reply:
Since the function is not implemented in this PR, I will modify it to another PR  #1341 .